Lab-3.ipynb

```diff
@@ -0,0 +1,728 @@
+      "text/plain": [
+       "<Figure size 720x720 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# random Gaussian data, and bad points\n",
+    "good = stats.norm.rvs(loc = 12., scale = 0.4, size = 100000)\n",
+    "bad = [10.0, 10.3, 2.1, 0.0, 0.0, 15.6, 22.3, 12.7]\n",
+    "bad = np.asarray(bad)\n",
+    "d = np.append(stats.norm.rvs(loc = 12., scale = 0.4, size = 100000), [10., 10.3, 2.1, 0., 0., 15.6, 22.3, 12.7])\n",
+    "\n",
+    "fig, ax = plt.subplots(1, 1, figsize=(10,10))\n",
+    "ax.set_title(\"Average Frequency Distribution of Temperature Readings\", fontsize=24)\n",
+    "ax.set_xlabel(\"Temperature Readings (Kelvin)\", fontsize=20)\n",
+    "ax.set_ylabel(\"Log(Frequency)\", fontsize=20)\n",
+    "\n",
+    "ax.hist(d,150, density=True)\n",
+    "plt.tick_params(labelsize = 24)\n",
+    "plt.yscale('log')\n",
+    "\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Here we have randomly generated \"good\" data in the form of a Gaussian distribution, and appended that data with an array of \"bad\" values. The continuous distribution in this plot follows a parabolic shape when we plot it as a semilog, while the random points of error that lie away from the mean do not follow any sort of continuous distribution."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We are trying to find a way mathematically to throw out as many bad data points as we can without throwing away the good ones. This means we are looking at how standard deviations make up the main distribution (the good data), then cut out any points that lie too far beyond this distribution (the bad data).\n",
+    "\n",
+    "Therefore, we can ask the following statistical question: \n",
+    "\n",
+    "For a signal-free background, how many standard deviations out from a mean of T=12 make up the continuous Gaussian-distributed data, the cutoff point for good data that we want to keep?"
```

Comment:
1.a.2: wording sounds off. Maybe "For a single-free background, what is the standard deviation value optimal for a cutoff point in a continuous Gaussian-distributed data set with a mean of 12 K?"